app/about_us/OurCrew.js

```diff
@@ -0,0 +1,102 @@
+import { Img } from "@chakra-ui/react";
+import styles from "./OurCrew.module.css";
+
+import image4 from "../../public/crew/image-victor-glover.png";
+import image5 from "../../public/crew/nasaCrewMember1.jpg";
+import image6 from "../../public/crew/nasaCrewMember2.webp";
+import Image from "next/image";
+import HanaLeePic from "../../public/crew/astronautHanaLee.webp";
+import PatelPic from "../../public/crew/astronautPatel.webp";
+export default function OurCrew() {
+  return (
+    <>
+      <h4 style={{ margin: "60px" }}>
```

Comment:
Don't get lazy here 😛 
It is usually forbidden to have in line styles with react.
They are very difficult to identify and maintain in the long run!
So please move the css out to the `.css` file.

Reply:
yes. i fix. i was just testing it worked the double curly brackets